README.md

```diff
@@ -21,6 +21,10 @@ npm install @coinbase/onchainkit
 pnpm add @coinbase/onchainkit
 ```
 
+### NextJS Pages Router
+
+If your app uses [NextJS Pages Router](https://nextjs.org/docs/pages), you'll need to ensure that Framekit is only called server-side (e.g., in `getServerSideProps` or `getStaticProps`). In [NextJS App Router](https://nextjs.org/docs/app) pages are React Server Components by default, so this is only a concern if you are calling from a client component.
```

Comment:
This is too much knowledge to share... I think it's important to be more explicit in our way to share import.

Docs shouldn't be the first solution, but instead the question I want to answear is, how to improve the library to be more flexible.